usr/include/pkcs11types.h

```diff
@@ -611,8 +611,8 @@ typedef CK_ULONG CK_ATTRIBUTE_TYPE;
 #define CKA_NSS                   (CKA_VENDOR_DEFINED | NSSCK_VENDOR_NSS)
 #define CKA_NSS_MOZILLA_CA_POLICY (CKA_NSS + 34)
 
-/* CK_ATTRIBUTE is a structure that includes the type, length
- * and value of an attribute */
+/* CK_ATTRIBUTE is a structure that includes the type, value
+ * and length of an attribute */
```

Comment:
Hm, well, yes, the order of the things in the comment was wrong., but is this change really related to the p11sak changes? I would prefer to have such 'cosmetic' changes in separate commits....

Reply:
This change already is in the output bug fixes + format changes commit an therefore separated from the from the actual p11sak changes

Reply:
But the commit message reads `[P11SAK] output bug fixes + format changes` where this change has nothing to do with P11SAK.

Reply:
Now in a standalone commit: textual changes only.